.github/hooks/pre-commit

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh -vx
+
+#
+# This pre-commit hook can work when you use `git add` if you run this command:
+#
+#   git config core.hooksPath .github/hooks
+#
+#
+
+FILES=$(git diff --cached --name-only --diff-filter=ACMR | grep '^source/class' | grep '.js$' | sed 's| |\\ |g')
+[ -z "$FILES" ] && exit 0
+
+# Prettify all selected files
+echo $FILES
+echo "$FILES" | xargs qx es6ify
```

Comment:
this assumes for qx to be installed globally ...

Reply:
What's about using a node script instead of the shell scrIpt? This should work for windows too.

Reply:
> this assumes for qx to be installed globally ...

this is for development of qooxdoo so we already assume a global compiler, and `npx qx` won't work because of our bootstrapping.

however, in your own projects, you would choose to whether to use `npx`, depending on how you added qooxdoo to your project